bears/vcs/git/GitCommitBear.py

```diff
@@ -151,6 +162,59 @@ def run(self,
             body,
             **self.get_issue_checks_metadata().filter_parameters(kwargs))
 
+    def check_revert(self, shortlog, body,
+                     allow_revert_commits_without_reason: bool = False):
+        """
+        Validates revert commits.
+
+        :param shortlog:
+            The shortlog message string.
+        :param body:
+            The body of the commit message of HEAD.
+        :param allow_revert_commits_without_reason:
+            Whether reason is needed for revert commits.
+        """
+        if not shortlog.startswith('Revert '):
+            return
+
+        shortlog_start_index = shortlog.find('"')
+        shortlog_end_index = shortlog.rfind('"')
+        reverted_shortlog = shortlog[shortlog_start_index +
```

Comment:
shortlog_start_index, shortlog_end_index and reverted_shortlog are confusing names in this context as, in other places of the code shortlog refers to the entire string including the subject and the message.